packages/@n8n/decorators/src/controller/route.ts

```diff
@@ -13,6 +13,8 @@ interface RouteOptions {
 	allowSkipMFA?: boolean;
 	/** When these options are set, calls to this endpoint are rate limited using the options */
 	rateLimit?: boolean | RateLimit;
+	/** When this flag is set to true, the endpoint is protected by API key */
+	apiKeyAuth?: boolean;
```

Comment:
If we decide to keep this part of the private API, this should be refactored. We have now two flags that concern the auth: `skipAuth` and `apiKeyAuth`. I think we should rather refactor this (in a separate PR, to be merged before this) to have a single option `auth: 'auth_token' | 'api_key' | 'skip'` with the default value of `auth_token`.